dumb.rb

```diff
@@ -2,6 +2,10 @@ class Dumb
   def bad_spacing
     { has: { with: { bad: :spacing } } }
   end
+
+  def more_dumb_stuff
+      x = 42
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `x`.<br>Style/IndentationWidth: Use 2 (not 4) spaces for indentation.